vuepress/docs/v7.0/tutorials/README.md

```diff
@@ -3,16 +3,17 @@ sidebarDepth: 2
 ---
 # Learning Paths
 
-::: tip Entando simplifies the development of modern apps:
+::: tip The Entando Platform simplifies and automates the DevOps lifecycle of applications orchestrated by Kubernetes.
 
-1. Built using modern JavaScript frameworks
-2. Backed by microservices deployed in containers, and
-3. Orchestrated by Kubernetes for fully automated DevOps lifecycles
-:::
 
-Entando supports full stack micro frontend and microservice architectures for codebases that are easier to understand, maintain, and debug across large, distributed teams, and comes with a private component repository that makes it easy for teams to share and reuse components.
+* Build end-to-end modularity into your application for codebases that are easier to maintain & debug.
```

Comment:
& -> and

Reply:
I think this was missed. I still see the &.